widget/tooltip.go

```diff
@@ -278,10 +280,13 @@ func (t *ToolTip) showingState(p image.Point) toolTipState {
 		sx, sy := t.content.PreferredSize()
 
 		position := p
-		if t.Position == TOOLTIP_POS_CURSOR_FOLLOW {
+		switch t.Position {
+		case TOOLTIP_POS_CURSOR_FOLLOW:
 			position = cp
-		} else if t.Position == TOOLTIP_POS_WIDGET {
+		case TOOLTIP_POS_WIDGET:
 			position = t.processWidgetPosition(parent.Rect)
+		case TOOLTIP_POS_SCREEN:
```

Comment:
I think we should change this to TOOLTIP_POS_ABSOLUTE.  My reasoning is that we may want to add "TOOLTIP_POS_SCREEN" that works like the widget one where you can tie it to different locations around the screen, (top left, top right, etc).

Reply:
I renamed it just now. I like the idea of TOOLTIP_POS_SCREEN :)

Reply:
Latest commits add TOOLTIP_POS_SCREEN too